data/presets/shop/optician.json

```diff
@@ -11,5 +11,8 @@
     "tags": {
         "shop": "optician"
     },
-    "name": "Optician"
+    "name": "Optician",
+    "aliases": [
+        "Eyewear Store"
```

Comment:
actually, are sunglasses etc supposed to be tagged as shop=optician if shop sells only sunglasses and no vision-correcting glasses?

Reply:
does search for Eyewear Store brings up also `shop=fashion_accessories` if these shops can be also described in this way?